hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
```

Comment:
@francisco-dlp not really for PCA - the reason I used soft rather than hard-thresholding (i.e. PCA) of the singular values was because it's easier to automate! @bm424 might know more.